spring-integration-core/src/main/java/org/springframework/integration/util/MessagingMethodInvokerHelper.java

```diff
@@ -979,7 +979,7 @@ private void setExclusiveTargetParameterType(TypeDescriptor targetParameterType,
 		}
 	}
 
-	private static class ParametersWrapper {
+	public static class ParametersWrapper {
```

Comment:
OK. I can live with that.

Ha-ha. Yes, it really doesn't point to such a violation. Only to the unused fields.